Source/cmake/GStreamerChecks.cmake

```
@@ -65,6 +65,14 @@ if (ENABLE_MEDIA_STREAM AND ENABLE_WEB_RTC)
         if (NOT OPENSSL_FOUND OR OPENSSL_VERSION VERSION_LESS "3.0.0")
             message(FATAL_ERROR "OpenSSL 3 is needed for ENABLE_WEB_RTC.")
         endif ()
+
+        if (USE_LIBRICE)
+            find_package(LibRiceIo 0.1.1 REQUIRED)
+            find_package(LibRiceProto 0.1.1 REQUIRED)
+            if (NOT LibRiceIo_FOUND OR NOT LibRiceProto_FOUND)
+                message(FATAL_ERROR "librice-{io,proto} is needed for USE_LIBRICE.")
+            endif ()
```

We do not need to manually produce an error when passing `REQUIRED` to the `find_package()` command; CMake will notify itself about the error. Sometimes we skip the `REQUIRED` flag in order to give a better error message and hinting about an `USE_*` or `ENABLE_*` option that can be toggled, which I think is the case here.


```suggestion
            find_package(LibRiceIo 0.1.1)
            find_package(LibRiceProto 0.1.1)
            if (NOT LibRiceIo_FOUND OR NOT LibRiceProto_FOUND)
                message(FATAL_ERROR "librice-{io,proto} is needed for USE_LIBRICE.")
            endif ()
```